src/components/layout/layout.js

```diff
@@ -121,14 +142,26 @@ const Layout = ({ children, type, interim_type, padding_top, no_login_signup })
             <Main padding_top={padding_top} is_static={is_static}>
                 {children}
             </Main>
-
             {show_cookie_banner && (
                 <CookieBanner
                     onAccept={onAccept}
                     onDecline={onDecline}
                     is_open={show_cookie_banner}
                 />
             )}
+            {is_livechat_interactive && (
+                <LiveChat
+                    id="gtm-deriv-livechat"
+                    onClick={() => {
+                        LC_API.open_chat_window()
+                    }}
+                    onMouseEnter={() => setLivechatHover(true)}
+                    onMouseLeave={() => setLivechatHover(false)}
+                >
+                    {is_livechat_hover ? <LiveChatHover /> : <LiveChatIC />}
```

Comment:
can this be handled with css? 🤔 would be faster and more interactive.

Reply:
i thought of that at first, but its a different svg structure

Reply:
wouldn't a parent element + display-none on hover for `LiveChatIC` (and reverse) for `LiveChatHover` work?

Reply:
actually nvm it's fast enough 👍